CHANGELOG.rst

```diff
@@ -16,6 +16,10 @@ Please See the `releases tab <https://github.com/openedx/xblock-lti-consumer/rel
 Unreleased
 ~~~~~~~~~~
 
+9.13.1 - 2025-01-15
+-------------------
+* Fix broken LTI 1.3 launch
```

Comment:
Could you make this a little more descriptive, so that it's clear what the bug was/what was fixed, please?